src/commands/BootstrapCommand.js

```diff
@@ -439,6 +441,7 @@ export default class BootstrapCommand extends Command {
               pkg.location,
               deps.map(({ dependency }) => dependency),
               this.npmConfig,
+              npmGlobalStyle,
```

Comment:
An assertion in [this test](https://github.com/lerna/lerna/blob/master/test/BootstrapCommand.js#L121) would be helpful:
```js
expect(NpmUtilities.installInDir).toBeCalledWith(
  expect.stringContaining("package-dirname"),
  expect.arrayContaining(["non-hoisted-dependency-name"]),
  expect.any(Object),
  true, // npmGlobalStyle
  expect.any(Function)
);
```

Of course, this means our `BootstrapCommand/basic` fixture (or a new, hoist-focused variant) needs a non-hoistable external dependency in one of its packages.